examples/plugins/src/js-benchmarking/README.md

```diff
@@ -0,0 +1,132 @@
+# benchmark js example
+
+🕵️️ **Code PushUp plugin to benchmark JS execution performance** 📊
+
+---
+
+The plugin analyzes a given suite name and creates benchmark audits.  
+It uses [tinybench](https://github.com/tinylibs/tinybench) under the hood.
+
+You can configure the plugin with the following options:
+
+- `targets` - files to load that export a suite
+- `tsconfig` - path to tsconfig file _(optional)_
+- `logs` - additional information _(optional)_
+
+## Getting started
+
+1. If you haven't already, install [@code-pushup/cli](../../../../packages/cli/README.md) and create a configuration file.
+
+2. Copy the [plugin source](./src/) as is into your project
+
+3. Add this plugin to the `plugins` array in your Code PushUp CLI config file (e.g. `code-pushup.config.js`).
+
+   Pass in the path on the directory to load the test suite form (relative to `process.cwd()`), for more options see [BenchmarkJsPluginOptions]().
+
+   ```js
+   import { join } from 'node:path';
+   import benchmarkJsPlugin from './benchmark-js.plugin';
+
+   export default {
+     // ...
+     plugins: [
+       // ...
+       await benchmarkJsPlugin({
+         targets: ['suites/score-report.ts'],
+       }),
+     ],
+   };
+   ```
+
+3.1. Create benchmark suite:
+
+```ts
+// typescript
+const suiteConfig = {
+  suiteName: 'glob',
+  targetImplementation: 'version-2',
+  cases: [
+    ['version-1', () => new Promise(resolve => setTimeout(resolve, 30))],
+    ['version-2', () => new Promise(resolve => setTimeout(resolve, 10))],
+    ['version-3', () => new Promise(resolve => setTimeout(resolve, 20))],
+  ],
+};
+```
```

Comment:
In Markdown, you have to use indentation to nest lists:

```suggestion
   1. Create benchmark suite:

      ```ts
      // typescript
      const suiteConfig = {
        suiteName: 'glob',
        targetImplementation: 'version-2',
        cases: [
          ['version-1', () => new Promise(resolve => setTimeout(resolve, 30))],
          ['version-2', () => new Promise(resolve => setTimeout(resolve, 10))],
          ['version-3', () => new Promise(resolve => setTimeout(resolve, 20))],
        ],
      };
      ```
```

### Before

![image](https://github.com/code-pushup/cli/assets/34691111/9d4b693e-e4ac-4326-9dd6-551895fc28ea)


### After

![image](https://github.com/code-pushup/cli/assets/34691111/2b9dc01a-907e-49d4-a4b7-d29fed3d35ea)

Reply:
Thanks for the detailed explanation. TBH I never got the second intention. :D